### ISSUE TITLE
Uninitialized field `opnorm` in JacVecOperator

### ISSUE THREAD

**Issue by danielwe** [SciML/DiffEqOperators.jl#519]
`opnorm` is left uninitialized here:

https://github.com/SciML/DiffEqOperators.jl/blob/4b1bd632f717aee8be6f26601b37322f11f6b94a/src/jacvec_operators.jl#L24-L33

Another instance of JuliaLang/julia#24943

**Comment by danielwe** [comment]
Same issue in vecjac_operators.jl:

https://github.com/SciML/DiffEqOperators.jl/blob/4b1bd632f717aee8be6f26601b37322f11f6b94a/src/vecjac_operators.jl#L23-L32

**Comment by ChrisRackauckas** [comment]
These should probably just be deprecated and deleted in favor of the ones that are more updated in SparseDiffTools.

**Comment by xtalax** [comment]
Are we sure that they are completely obselete? I'd prefer to leave the operators up for now until we have stencil interfaces in MOL for the high point count problems and warn of future deprecation, with links to successor packages

**Comment by ChrisRackauckas** [comment]
Yes, these ones are not robust to alternative tags like SparseDiffTools is, so pretty much any use case should use that. There's a few things in here that aren't in there to match the AbstractSciMLOperator interface, but what we probably need to do is just make that interface package a lean package and then let SparseDiffTools extend that, add the one function, and then there's no reason for the operators here.
